### PR TITLE
Quick patch to flag lines that are different to as expected in tests.

### PR DESCRIPTION
**Pull request by Ekrekr**

### CODE REVIEW

File: testing/runner.ts [hunk 1]
```diff
@@ -96,7 +97,16 @@ export class Runner {
               console.error(`\n    Actual:\n`);
               console.error(indent(JSON.stringify(result.err.actual, null, 4), 8));
             }
-            console.error("\n");
+            console.error(`\n    Overall diff:`);
```
Comment (lewish) [inline testing/runner.ts]: In some cases, either expected or actual is missing, can you wrap this in an if statement?

File: testing/runner.ts [hunk 2]
```diff
@@ -96,7 +97,16 @@ export class Runner {
...
+            Diff.diffLines(
+              indent(JSON.stringify(result.err.expected)),
```
Comment (lewish) [inline testing/runner.ts]: I think you probably want to indent these after?

File: testing/runner.ts (hunk 1 above)
Comment (Ekrekr) [reply testing/runner.ts]: Ah good catch

File: testing/runner.ts (hunk 2 above)
Comment (Ekrekr) [reply testing/runner.ts]: Meh, either works

**Pull request merged**